pkgs/desktops/gnome/apps/gnome-boxes/default.nix

```diff
@@ -120,7 +121,7 @@ stdenv.mkDerivation rec {
   ];
 
   preFixup = ''
-    gappsWrapperArgs+=(--prefix PATH : "${lib.makeBinPath [ mtools cdrkit libcdio ]}")
+    gappsWrapperArgs+=(--prefix PATH : "${lib.makeBinPath [ mtools cdrkit libcdio qemu ]}")
```

Comment:
Maybe qemu-utils would be sufficient.

Reply:
Yep it is. I'll update my PR this evening